config_example.json

```diff
@@ -7,5 +7,24 @@
     "x86": "android-x86-opt",
     "armv7_v15": "android-api-15-opt"
   },
+  "task": {
+    "name": "Google Play Publisher",
+    "description": "Publishes Aurora builds to Google Play Store",
+    "owner": "r@m.c",
+    "treeherder": {
+      "platform": "Android",
```

Comment:
Yeah, I initially started to use "android-api-15", but as this task is about publishing every architecture available, I believe something general like "Android" fits the scope better.